docs/install-and-setup.md

```diff
@@ -220,7 +220,7 @@ transition once we default to these new completions.
 
 !!! info "Why are the improved completions not the default?"
 
-    To get the improved completions it is required to set the `COMPLETION` env
+    To get the improved completions it is required to set the `COMPLETE` env
```

Comment:
nit: (unrelated to this PR), I think this should say "To generate improved completion script, ... execute `jj` with `COMPLETE` env variable" or something. With the original wording, I would add `COMPLETE=` to my `~/.zshrc`.

Reply:
Good point. I reworded it a bit more. Hopefully it's clearer.